spec/recurly/client_spec.rb

```diff
@@ -54,6 +54,15 @@
         expect(net_http).to receive(:request).and_return(response)
         _account = subject.get_account(account_id: "code-benjamin-du-monde")
       end
+
+      it "should include custom headers in each request" do
+        expected = hash_including("Custom-Header" => "custom-value", "Accept" => /application\/vnd\.recurly/, "Content-Type" => "application/json", "User-Agent" => /Recurly\//)
```

Comment:
Need to assert this

Reply:
Yeah, I based this spec off of the one above it and neither is actually asserting the `expected`. I'll update it